### PR TITLE
Soften cache directory creation in install docs

### PR DESCRIPTION
**Pull request by stsievert**
**What does this PR do?**

Functionally, it changes "need to create cache directory" to "can create a cache directory.

### CODE REVIEW

**Comment by jfowkes** [comment]
Thanks @stsievert, @lindonroberts could you review as you implemented the cache handling?

File: docs/install.rst [hunk 1]
```diff
@@ -126,31 +126,19 @@ For easy installation, use `pip <http://www.pip-installer.org/>`_:
...
-
-    export PYCUTEST_CACHE="/path/to/pycutest_cache"
-
-If you do not set this environment variable, then PyCUTEst will create a cache folder of compiled problems inside your current working directory.
+    $ export PYCUTEST_CACHE="/path/to/pycutest_cache"
 
 Note that you can uninstall PyCUTEst as follows:
 
- .. code-block:: bash
+.. code-block:: bash
```
Comment (jfowkes) [inline docs/install.rst:142]:
```suggestion
 .. code-block:: bash
```

File: docs/install.rst [hunk 2]
```diff
@@ -126,31 +126,19 @@ For easy installation, use `pip <http://www.pip-installer.org/>`_:
...
 
-    $ pip uninstall pycutest
+   $ pip uninstall pycutest
```
Comment (jfowkes) [inline docs/install.rst:144]:
```suggestion
    $ pip uninstall pycutest
```

**Review by lindonroberts — APPROVED** [review]
Looks good to me, this description of the cache path is correct

**Pull request merged**